packages/base/src/3dview/mainview.tsx

```diff
@@ -1110,6 +1122,23 @@ export class MainView extends React.Component<IProps, IStates> {
         this._updateClipping();
       }
     }
+
+    if (change.key === 'wireframe') {
+      const wireframeEnabled = change.newValue as boolean | undefined;
+
+      if (wireframeEnabled !== undefined) {
+        this.setState({ wireframe: wireframeEnabled }, () => {
+          this._scene.traverse(child => {
```

Comment:
Can you try only setting wireframe to true/false on the materials of the group `this._meshGroup`? That way you would not apply the wireframe effect on the clipplane control/pointers etc. We suppose only want the actual objects we're visualizing to be wireframed.

Reply:
Done in [`18a2c7f`](https://github.com/jupytercad/JupyterCAD/pull/390/commits/18a2c7ffb077606ef85f8d0e2ee71c05074893a3)